### DISCUSSION TITLE
Customize FormMessages

### DISCUSSION THREAD

**Discussion by jmail** [keycloak/keycloak#29059] [Q&A]
How can I get separate message for this use case? https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/authentication/AuthenticationProcessor.java#L746

**Comment by codespearhead** [comment]
What do you mean? What are you trying to accomplish?

**Reply by jmail** [reply]
We got one error message for:
- INVALID_USER https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/authentication/AuthenticationProcessor.java#L736-L738
- and 
- USER_TEMPORARILY_DISABLED https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/authentication/AuthenticationProcessor.java#L744-L746
cases. 

As we are really using temporarily disabled we want to be able to inform user that his account is disabled only for time. Right now the Form message is INVALID_USER for both cases which is not true in case of temp disable. I hope it is clear enough. We just want to have different message key to inform user about temp disable

**Reply by codespearhead** [reply]
In that case, this is a duplicate of #26293 . How about you close this discussion and we carry on over there?
